### PR TITLE
Ganesh dev

### PR DESCRIPTION
**Pull request by ganeshgore**
Pushed FPGA22 design with SKY130_FD_SC_HD

### CODE REVIEW

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info [hunk 1]
```diff
@@ -0,0 +1,55 @@
+commit 48b2bff0d909e2c6d0740d2a5386123eb238349f
+Author: tangxifan <tangxifan@gmail.com>
```
Comment (tangxifan) [inline FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info:2]: No sure if this file should be uploaded

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater [hunk 2]
```diff
@@ -0,0 +1 @@
+/research/ece/lnis/CAD_TOOLS/DKITS/skywater
```
Comment (tangxifan) [inline FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater:1]: Symbolic link should not be uploaded.

**Review by tangxifan — CHANGES_REQUESTED** [review]

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info (hunk 1 above)
Comment (ganeshgore) [reply FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info:2]: This is to log which commit version has been used for netlist generation

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater (hunk 2 above)
Comment (ganeshgore) [reply FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater:1]: You can remove this, or may be add in .gitignore

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info (hunk 1 above)
Comment (tangxifan) [reply FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info:2]: O.K. Let's keep it for now. But users should use the master of OpenFPGA all the times. We ensure backward compatibility.

File: FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater (hunk 2 above)
Comment (tangxifan) [reply FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater:1]: I can remove but I will have to modify (pollute) your branch. It is up to you.

Comment (ganeshgore) [reply FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_task/skywater:1]: I will push update on this branch in sometime

**Review by tangxifan — APPROVED** [review]

**Pull request merged**